webpack.main.config.ts

```diff
@@ -60,6 +62,19 @@ export const mainBaseConfig: Configuration = {
 				process.env.GITHUB_SHA ?? process.env.BUILDKITE_COMMIT ?? undefined
 			),
 		} ),
+		new CopyWebpackPlugin( {
+			patterns: [
+				// Copy about-menu.html into the main output directory
+				{
+					from: path.resolve( __dirname, 'src/about-menu/about-menu.html' ),
+					to: path.resolve( __dirname, '.webpack/main/menu' ),
+				},
+				{
+					from: path.resolve( __dirname, 'src/about-menu/studio-app-icon.png' ),
+					to: path.resolve( __dirname, '.webpack/main/menu' ),
```

Comment:
Let's remove the slashes, so it can work in different operating systems.

```suggestion
					from: path.resolve( __dirname, 'src', 'about-menu', 'about-menu.html' ),
					to: path.resolve( __dirname, '.webpack', 'main', 'menu' ),
				},
				{
					from: path.resolve( __dirname, 'src', 'about-menu', 'studio-app-icon.png' ),
					to: path.resolve( __dirname, '.webpack', 'main', 'menu' ),
```

Reply:
Sounds good, thanks for the suggestion. Let's keep it as is then 👍